Tests/PayPalMessagesTests/PayPalMessageLoggerTests.swift

```diff
@@ -366,8 +411,16 @@ final class PayPalMessageLoggerTests: XCTestCase {
 
     private func assert(payload: [String: Any], equals expectedPayload: [String: Any]) {
         var data = payload
+        var expected = expectedPayload
```

Comment:
Is there a specific reasons we're renaming this variable?

Reply:
`expectedPayload` is a constant so I created a new var in order to remove `id` and `time` from the payload after verifying they were there because they are generated uniquely each time, then use that to assert.